runner/init.sls

```diff
@@ -0,0 +1,82 @@
+# This state installs a gitlab-runner and optionally also deploys its configuration (config.toml)
```

Comment:
Another nitpick: As this is now a single file again, we could consider moving it back to `runner.sls` instead of `runner/init.sls`...